donkeycar/parts/camera.py

```diff
@@ -22,6 +22,32 @@ def __init__(self, image_w=160, image_h=120, image_d=3, framerate=20, vflip=Fals
         self.camera.framerate = framerate
         self.camera.vflip = vflip
         self.camera.hflip = hflip
+        self.camera.awb_mode = awb_mode
```

Comment:
Then here, you would iterate through all the possible parameters of the camera using `get()` and providing a default values. Hence only required overwrites would need to be in the dictionary.